app/index.html

```diff
@@ -16,6 +16,7 @@
 <link rel="icon" sizes="512x512" href="img/icons/512.png">
 <meta name="application-name" content="Foxbox Client">
 <title>Foxbox Client</title>
+<script type="text/javascript" src="cordova.js"></script>
```

Comment:
This is needed for the Cordova build; I added an empty `app/cordova.js` file to the repo so that it won't error when using `gulp` to serve the app on http://localhost:8000/. We should do this in a fancier way eventually, but I think for now it does the trick. :)